Source/Core/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_subdirectory(AudioCommon)
 add_subdirectory(Common)
 add_subdirectory(Core)
 add_subdirectory(DiscIO)
-add_subdirectory(DolphinWX)
+if(NOT DISABLE_WX)
+  add_subdirectory(DolphinWX)
```

Comment:
Tab here please, or change the one below to 2 spaces instead to use the recommended style everywhere in that file.